src/nu/validator/checker/UnsupportedFeatureChecker.java

```diff
@@ -11,8 +11,6 @@
      */
     @Override public void startElement(String uri, String localName,
             String qName, Attributes atts) throws SAXException {
-        boolean followW3Cspec = "1".equals(System.getProperty("nu.validator.servlet.follow-w3c-spec")) ? true
-            : false;
         if ("http://www.w3.org/1999/xhtml" != uri) {
```

Comment:
> This same URL appears in at least 32 different files. Maybe we should consider creating a constants class to hold it? Assuming there are enough strings with similar frequency to make it worthwhile.

A grep shows there's 163 instances of it in code in the tree so yeah it might make some sense.